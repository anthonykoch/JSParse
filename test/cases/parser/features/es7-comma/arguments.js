es7_trailing(a,)
