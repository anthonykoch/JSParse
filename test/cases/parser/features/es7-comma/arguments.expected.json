---
name: ES7 Comma - function
description: parses allows trailing comma in function
options:
  context:
    es7_trailing_comma: true
---

[
  {
    "type": "ExpressionStatement",
    "start": 0,
    "end": 16,
    "expression": {
      "type": "CallExpression",
      "callee": {
        "type": "Identifier",
        "name": "es7_trailing",
        "start": 0,
        "end": 12
      },
      "start": 0,
      "end": 16,
      "arguments": [
        {
          "type": "Identifier",
          "name": "a",
          "start": 13,
          "end": 14
        }
      ]
    }
  }
]
